rocAL/rocAL/include/augmentations/color_augmentations/node_color_twist.h

```diff
@@ -24,28 +24,23 @@ THE SOFTWARE.
 #include "node.h"
 #include "parameter_factory.h"
 #include "parameter_vx.h"
-#include "graph.h"
 
-class ColorTwistBatchNode : public Node
-{
+class ColorTwistNode : public Node {
 public:
-    ColorTwistBatchNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
-    ColorTwistBatchNode() = delete;
+    ColorTwistNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
+    ColorTwistNode() = delete;
     void init(float alpha, float beta, float hue, float sat);
-    void init(FloatParam *alpha, FloatParam *beta, FloatParam *hue, FloatParam *sat);
-
+    void init(FloatParam *alpha_param, FloatParam *beta_param, FloatParam *hue_param, FloatParam *sat_param);
 protected:
     void create_node() override;
     void update_node() override;
 private:
-
     ParameterVX<float> _alpha;
```

Comment:
Please check if is better to add a _vx suffix to ParameterVX type variables

Reply:
That is not required in these files, we can add that in future new augmentations